### PR TITLE
Update addon via `ember init`

### PR DESCRIPTION
**Pull request by elwayman02**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,41 +7,40 @@
...
+    "start": "ember server",
     "test": "ember try:testall"
   },
   "repository": "https://github.com/jmurphyau/ember-truth-helpers",
   "engines": {
     "node": ">= 0.10.0",
-    "ember-cli-app-version": "0.4.0"
+    "ember-cli-app-version": "1.0.0"
   },
-  "author": "James Murphy",
```
Comment (rwjblue) [inline package.json]: Please bring back.

**Comment by rwjblue** [comment]
FYI - This is the master ember-cli app blueprint (which I think js good, just mentioning).

File: package.json (hunk 1 above)
Comment (elwayman02) [inline package.json]: whoops xD

**Comment by elwayman02** [comment]
Yea, I've been npm linking to master for a while haha

**Comment by rwjblue** [comment]
Thanks!

**Pull request merged**